src/Components/Showcase/Showcase.tsx

```diff
@@ -1,20 +1,28 @@
 import './showcase.scss';
+import message from '@/images/Homepage/message.webp';
 import leftImage from '@/images/Homepage/homepageLeft.svg';
 import rightImage from '@/images/Homepage/homepageRight.svg';
 import titleImage from '@/images/Homepage/titleImg.svg';
-import arrow from '@/images/Homepage/arrow.svg';
+import arrow from '@/images/Homepage/arrow.png';
 
-const Showcase = () => {
+import { useNavigate } from 'react-router-dom';
+
+const Showcase: React.FC = () => {
+  const navigate = useNavigate();
+
+  const handleNavigate = () => {
+    navigate('/register');
```

Comment:
```suggestion
    navigate(paths.REGISTER);
```

Reply:
For reusing paths I created additional file with import paths from "./paths";


type MenuLinkConfig = {
    path: string;
    name: string;
  };
  
  const menuLinksConfigs: MenuLinkConfig[] = [
    { path: paths.HOME, name: 'Home' },
    { path: paths.CASE, name: 'Case Studies' },
    { path: paths.PRICE, name: 'Pricing' },
    { path: paths.ABOUT, name: 'About' },
    { path: paths.BLOG, name: 'Blog' },
  ];
. Now I can change  navigate(paths.REGISTER)